evap/student/templates/student_index.html

```diff
@@ -65,10 +65,10 @@
                                         <span class="badge badge-secondary">{{ course.type }}</span>
                                         {% if course in voted_courses %}
                                             {% if course.state == 'in_evaluation' %}
-                                                <span class="badge badge-success" data-toggle="tooltip" data-placement="top" title="{% trans 'You already evaluated this course.' %}"><span class="fa fa-smile-o" aria-hidden="true"></span></span>
+                                                <span class="text text-success" data-toggle="tooltip" data-placement="top" title="{% trans 'You already evaluated this course.' %}"><span class="far fa-smile" aria-hidden="true"></span></span>
                                             {% endif %}
                                         {% elif course.state == 'evaluated' or course.state == 'reviewed' or course.state == 'published' %}
-                                                <span class="badge badge-primary" data-toggle="tooltip" data-placement="top" title="{% trans 'You did not evaluate this course.' %}"><span class="fa fa-frown-o" aria-hidden="true"></span></span>
+                                                <span class="text text-primary" data-toggle="tooltip" data-placement="top" title="{% trans 'You did not evaluate this course.' %}"><span class="far fa-frown" aria-hidden="true"></span></span>
```

Comment:
if that's not font-awesome-related, can you move that to the other commits?

Reply:
it's related because the icons don't look good otherwise. i can still move it somewhere else if you tell me where exactly you would prefer that. i don't know where you want to have something if you say "to other commits" and i don't want to move it twice.

Reply:
no then it's fine. looked like a bootstrap thing to me.